src/Oxpecker/ModelBinder.fs

```diff
@@ -328,7 +291,4 @@ type ModelBinder(?options: ModelBinderOptions) =
         /// It will try to match each property of 'T with a key from the data dictionary and parse the associated value to the value of 'T's property.
         /// </summary>
         member this.Bind<'T>(data) =
-            let model = Activator.CreateInstance<'T>()
-            match ModelParser.parseModel<'T> model options.CultureInfo (Dictionary data) with
-            | Ok value -> value
-            | Error msg -> failwith msg
+            ModelParser.parseModel<'T> options.CultureInfo (RawValues(Dictionary data))
```

Comment:
It's better to create some wrapper around data (that allows indexing and enumeration), rather than rebuild dictionary right from the start